nss-day-cohort-47/tabloidfullstack-bingo
# Hp add categories

# Description
Wrote code to give the ability to add a category.

## Type of change
Please delete options that are not relevant.
-
# Testing Instructions
Log in to the app and navigate to categories page, select the Add Category button and fill out the form. Select save and you should be taken back to the list of categories showing the most recent at the bottom.
# Checklist:
